hope.yaml

```diff
@@ -14,19 +14,19 @@ nodes:
   # There should be three of these.
   # Any Kubernetes-Role nodes can either have a direct host, or name a
   #   hypervisor from which their IP Address can be fetched.
-  - name: home-master-01
+  - name: test-master-01
```

Comment:
Not for now, but might eventually be worth having a dedicated hope.yaml file for tests, rather than using the documentation file as a test file as well.